docs/book/src/topics/prerequisites.md

```diff
@@ -0,0 +1,51 @@
+# Prerequisites
+
+## Requirements
+
+- [helm](https://helm.sh/)
+- [CAPI management cluster](https://cluster-api.sigs.k8s.io/).
+    - Features `EXP_CLUSTER_RESOURCE_SET` and `CLUSTER_TOPOLOGY` must be enabled.
+    - It is recommend to use `KUBE_VERSION` >= 1.26.3.
+    - [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start.html?highlight=clusterctl#install-clusterctl).
+
+## Installation
+
+As this project is still in experimental mode, we recommend you start by installing the provider into a kind development cluster specific for this purpose, so you can interact with it in a safe test environment. **CAUTION: while features of this project are marked as experimental, you could experience unexpected failures**.
+
+### Create your local cluster
+
+1. Start by adding the helm repositories that are required to proceed with the installation.
```

Comment:
Makes sense, I will add `just` here. I wanted to also document the process explicitly just because sometimes you like to have the commands easily accessible in the docs.